routes/post.py

```diff
@@ -62,5 +62,4 @@ def post(self):
 
 api.add_resource(GetAllPosts, '/api/posts')
 api.add_resource(AddPost, '/api/posts')
-api.add_resource(AddPost, '/api/posts')
```

Comment:
Do we know how this line was duplicated?

Reply:
yeah i think i might've had a merge conflict from another branch